catalog/iam-policies.yaml

```diff
@@ -1,12 +1,11 @@
-- sid: "DenyCreatingIAMUsers"
+- sid: "DenyIAMCreatingUsers"
```

Comment:
if we want to follow this pattern:

Deny -> Resource -> Actions

then need to update other SIDs in this PR

Reply:
ok, will do